.gitignore

```diff
@@ -6,9 +6,7 @@ target
 
 /.settings/
 /.project
-/.DS_Store
-/spec/.DS_Store
-/api/.DS_Store
```

Comment:
Not sure what exactly these files are (I think they are Mac related and I don't use Mac) but shouldn't we keep them?

Reply:
Ah, this excludes all .DS_Store files, disregard my comment